### PR TITLE
Clean up user data storage

### PR DESCRIPTION
**Pull request by evulins**
Clean code and add tests for checkbox

### CODE REVIEW

File: cypress/integration/login_page.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-describe("Problem Detailed Page", () => {
+describe("Login Page", () => {
```
Comment (bazelajoanna) [inline cypress/integration/login_page.js:1]: 🙈

File: cypress/integration/login_page.js [hunk 2]
```diff
@@ -17,4 +17,31 @@ describe("Problem Detailed Page", () => {
...
     cy.contains("Błędne hasło");
   });
+
+  xit("doesn't logout user after one day, when they clicked 'Remember me' checbox", () => {
+    cy.get('input[name="email"]').type("kasia@tc.io");
+    cy.get('input[name="password"]').type("!Pass112233");
+    cy.get('[type="checkbox"]').check({ force: true });
+
+    cy.get("form").submit();
+    const twoDays = 60 * 60 * 24 * 1000 * 2;
```
Comment (bazelajoanna) [inline cypress/integration/login_page.js:27]: 👩‍🎓 ✔️

File: src/auth/utils.tsx [hunk 3]
```diff
@@ -3,14 +3,15 @@ import { UserTokenWithExpiry } from "./interface";
...
 ): void => {
-  const now = new Date();
   const userToken = {
-    token: token,
-    expiry: now.getTime() + ttl,
+    token,
+    expiry,
   };
 
+  console.log(expiry);
```
Comment (bazelajoanna) [inline src/auth/utils.tsx]: leftover? ;)

File: src/auth/AuthProvider.tsx [hunk 4]
```diff
@@ -1,17 +1,18 @@
...
 import { useNavigate } from "react-router-dom";
+import { addDays } from "date-fns/esm";
+import addMonths from "date-fns/addMonths";
 
-import { LOGIN, Login } from "../components/pages/LoginPage/interface";
-import { AuthContextType, AuthProps } from "./interface";
+import { LOGIN, Login, User } from "../components/pages/LoginPage/interface";
+import { AuthContextType, AuthProps, UserTokenWithExpiry } from "./interface";
 import { getUserDataFromStorage, setUserDataInStorage } from "./utils";
+import { addHours, addMinutes } from "date-fns";
```
Comment (bazelajoanna) [inline src/auth/AuthProvider.tsx]: Could this line be removed? Or will we need it?

**Review by bazelajoanna — COMMENTED** [review]
Great success! 🎉  Just two tiny questions :)

File: src/auth/AuthProvider.tsx (hunk 4 above)
Comment (evulins) [reply src/auth/AuthProvider.tsx]: I saw all your comments after I push the last commit. It should be ok now :)

**Pull request merged**